media-sound/pulseaudio-daemon/pulseaudio-daemon-16.1-r7.ebuild

```diff
@@ -248,6 +248,11 @@ src_configure() {
 src_install() {
 	meson_src_install
 
+	# qpaeq moved to media-sound/qpaeq
```

Comment:
We'll have to add a blocker on media-sound/qpaeq and vice-versa in media-sound/qpaeq on <.../pulseaudio-daemon-16.1-r7.

Reply:
New `media-sound/qpaeq` dependends on `>=media-sound/pulseaudio-daemon-16.1-r7[dbus,fftw]`
I do not think that extra blocker is required, as qpaeq is via `optfeature`, correct?

Reply:
You are correct, added the blocker now.